lib/msal-browser/src/interaction_client/PopupClient.ts

```diff
@@ -28,11 +28,15 @@ export class PopupClient extends StandardInteractionClient {
             // asyncPopups flag is true. Acquires token without first opening popup. Popup will be opened later asynchronously.
             if (this.config.system.asyncPopups) {
                 this.logger.verbose("asyncPopups set to true, acquiring token");
+                // Passes on popup position and dimensions if in request
+                if (request.popupConfiguration) {
```

Comment:
You use the pattern "if popup config then do with popup config, else do without popup config" throughout the code.  You might want to consider adding a default popup config that's always used to eliminate the if/elses and just use the do with popup config approach.  If the user passes in values, you can overwrite the default popup config.

Reply:
Thanks for the suggestion. Updated the approach.